packages/client/wallets/aa/src/api/CrossmintWalletService.ts

```diff
@@ -8,36 +8,31 @@ import { BaseCrossmintService } from "./BaseCrossmintService";
 export { EVMBlockchainIncludingTestnet } from "@crossmint/common-sdk-base";
 
 export class CrossmintWalletService extends BaseCrossmintService {
```

Comment:
Check https://github.com/Paella-Labs/crossbit-main/pull/13736/files for more context on these changes